docs/sources/introduction/_index.md

```diff
@@ -0,0 +1,16 @@
++++
+title = "Introduction to Grafana"
+weight = 5
+aliases = ["/docs/grafana/latest/guides/what-is-grafana"]
++++
+
+# Introduction to Grafana
+
+Grafana is a complete observability stack that allows you to monitor and analyze mertrics, logs and traces. When monitoring data, an understanding of relevant data and the relationship between them is important. Grafana allows you to seamlessly go from metrics to logs, metrics to traces, and from logs to traces. For more information, refer to [Grafana overview](https://grafana.com/grafana/). Our observability stack has the following products and components.
```

Comment:
Can we incorporate any of this content from the Product page? "Grafana allows you to query, visualize, alert on and understand your metrics no matter where they are stored. Create, explore, and share beautiful dashboards with your team and foster a data driven culture."

Reply:
maybe we should say "data no matter where it is stored" to better incorporate logs and traces?

Reply:
I have added your suggestion.